test/util.js

```diff
@@ -1,6 +1,6 @@
-import test from 'ava';
-import util from '../util';
-import stubs from './helpers/stubs';
+const test = require('ava');
+const util = require('../util');
+const stubs = require('./helpers/stubs');
```

Comment:
These should not be changed but instead the the rest of the module should be turned into ESM (only).

Reply:
> move the AVA upgrade into a separate PR (because it means converting the module into ESM)

Given that (as far as I can tell, anyway) this does *work*, couldn't converting everything to ESM come later?

Reply:
I'd rather not have to change everything to CJS and then back to ESM, but I think it's ok